kie-wb-tests/kie-wb-tests-gui/src/test/java/org/kie/wb/selenium/ui/LoadAllPerspectivesIntegrationTest.java

```diff
@@ -45,11 +46,10 @@ public void allPerspectivesCanBeLoaded() {
                 AbstractPerspective perspective = home.getNavbar().navigateTo( _p );
 
                 //ProjectAuthoring will direct to ProjectLibrary if there are no Projects in the workbench
-                if ( perspective instanceof ProjectAuthoringPerspective ) {
-                    if ( !perspective.isDisplayed() ) {
-                        LOG.info( "ProjectAuthoringPerspective not displayed. Trying fallback ProjectLibraryPerspective.." );
-                        _p = Persp.PROJECT_LIBRARY;
-                        perspective = home.getNavbar().navigateTo( _p );
+                if (perspective instanceof ProjectAuthoringPerspective) {
+                    if (((ProjectAuthoringPerspective) perspective).isAuthoringDisabled()) {
```

Comment:
I don't understand why we need both isDisplayed and isAuthoringDisplayed in the ProjectAuthoringPerspective. Can't we collapse it to one method somehow?